cmd/commands/runtime.go

```diff
@@ -201,16 +206,20 @@ func runtimeUpgradeCommandPreRunHandler(cmd *cobra.Command, args []string, opts
 		return err
 	}
 
-	isManaged, err := isRuntimeManaged(ctx, opts.RuntimeName)
+	rt, err := getRuntime(ctx, opts.RuntimeName)
 	handleCliStep(reporter.UpgradeStepPreCheckIsManagedRuntime, "Checking if runtime is hosted", err, true, false)
 	if err != nil {
 		return err
 	}
 
-	if isManaged {
+	if rt.Managed {
 		return fmt.Errorf("manual upgrades are not allowed for hosted runtimes and are managed by Codefresh operational team")
 	}
 
+	if rt.AccessMode == platmodel.AccessModeTunnel {
+		opts.installFeatures = append(opts.installFeatures, runtime.InstallFeatureIngressless)
+	}
```

Comment:
if runtime (data from platform) has AccessMode of `TUNNEL` -> add the "ingressless" installFeature to the options